pkg/commands/kustfile/kustomizationfile.go

```diff
@@ -292,3 +286,11 @@ func marshalField(field string, kustomization *types.Kustomization) ([]byte, err
 
 	return yaml.Marshal(k)
 }
+
+func isEmpty(v reflect.Value) bool {
+	// If v is a pointer type
+	if strings.HasPrefix(v.Type().String(), "*") {
```

Comment:
This seems to be hacky.
I guess you can check if `v.Type().Kind() == reflect.Ptr`. Please let me know if that doesn't work.

Reply:
This is exactly what I want. Thank you so~~~ much. It works.